docs/topics/db/optimization.txt

```diff
@@ -255,8 +261,8 @@ But:
 
 .. _overuse_of_count_and_exists:
 
-Don't overuse ``count()`` and ``exists()``
-------------------------------------------
+Don't overuse ``contains()``, ``count()`` and ``exists()``
```

Comment:
Comma is missing before 'or'/'and' (see https://en.wikipedia.org/wiki/Serial_comma).